context/app/api/vitessce.py

```diff
@@ -320,8 +333,35 @@ def _build_vitessce_conf(self):
             found_images = _get_matches(file_paths_found, files[0]["rel_path"])
             # Do not show IMS images that are in the "/separate/" folder.
             no_ims_separate_images = _exclude_matches(found_images, r"/separate/")
-            layer = self._build_multi_file_image_layer_conf(no_ims_separate_images)
-            conf["layers"] = [layer]
+            # File paths are like path/to/HybCycle_N/MMStack_PosM.ome.tif and we want
+            # each conf to be all images across a given PosM, each image named HybCycle_N.
+            # This is unique to seqFish.
+            if "seqFish" in self.entity["data_types"]:
+                # Get all files grouped by PosN names.
+                images_by_pos = _group_by_file_name(found_images)
+                # Get Hybridization per paths grouped by Pos.
+                hyb_cycles_per_pos = [
+                    sorted([Path(image).parts[-2] for image in images])
+                    for images in images_by_pos
+                ]
```

Comment:
This is one of the places where we're really expecting a very particular structure, and I'd be more comfortable if there were validation up-front that that structure holds. I also think we should have some kind of testing about the viewconfs that are produced.

Reply:
Would a regex work?  It bypasses the "structure" a bit but it makes sure that what I need is there.

Something like
```python
re.search(r"HybCycle_[0-9]", 'output-pyramids/HybCycle_4/MMStack_Pos25.ome.tiff')[0]
# HybCycle_4
```

Reply:
I mean more generally, would regexes work as a way of validating?  We could also do something more like a `match` with the directory structure.

Reply:
@mccalluc Do you have an opinion here about what sort of validation is done? Or is this more of a "something is better than nothing" situation?

Reply:
@ilan-gold : Regexes could work. I'm really not sure what part is fixed and which is variable, but perhaps checking that every path matches:`output-pyramids/HybCycle_\d+/MMStack_Pos\d+\.ome\.tiff`? Let me look at the latest changes...